internal/btf/testdata/fuzz/FuzzExtInfo/72534f53bd90cb52a017013499b11511535c1295bf0e22f856148c02454c323e

```diff
@@ -0,0 +1,3 @@
+go test fuzz v1
```

Comment:
No cycles to look into this at the moment, but could we automatically test against known crashers in CI? That would prevent these from rotting in the future.

Reply:
I think that is the default behaviour:

```
$ go1.18beta1 test -exec sudo -v .
...
=== RUN   FuzzExtInfo
=== RUN   FuzzExtInfo/seed#0
=== RUN   FuzzExtInfo/72534f53bd90cb52a017013499b11511535c1295bf0e22f856148c02454c323e
=== RUN   FuzzExtInfo/a87a26efa64ed50b598ae8e333301d57d5f234527730f042d68ccc736e90c9fa
--- PASS: FuzzExtInfo (0.00s)
    --- PASS: FuzzExtInfo/seed#0 (0.00s)
    --- PASS: FuzzExtInfo/72534f53bd90cb52a017013499b11511535c1295bf0e22f856148c02454c323e (0.00s)
    --- PASS: FuzzExtInfo/a87a26efa64ed50b598ae8e333301d57d5f234527730f042d68ccc736e90c9fa (0.00s)
```

Reply:
I thought it might be, very neat!